hw/xbox/nv2a_shaders.c

```diff
@@ -280,10 +290,42 @@ static QString* generate_fixed_function(const ShaderState state,
     return s;
 }
 
+static GLuint create_and_compile_gl_shader(GLenum gl_type,
```

Comment:
I agree about gl_shader_type because the GL name was GLuint shaderType.
However, the function name itself should at least show that it's a pure GL helper function IMHO.
I'd accept compile_gl_shader() or create_gl_shader() but I feel compile_shader is not explanatory enough. Ideally all our functions / variable-names should show if they belong to the graphics API. Hopefully that will make it easier to port to something such as vulkan, D3D or software rendering later.